Sources/Extensions/Collection+Safe.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Collection+Safe.swift
+//
+//  Copyright © 220 Rosberry. All rights reserved.
+//
+
+import Foundation
+
+extension MutableCollection {
+
+    /// Returns the element at the specified index if it is within bounds, otherwise nil.
+    subscript (safe index: Index) -> Element? {
```

Comment:
This extension can be improved if Index is Int -- compare `count` and `index` instead of `contains` usage

Reply:
No, in the dictionary case:
[1: "One", 2:"Two", 4:"Four"] collection count is 3 but keys does not contains 3

Reply:
`[Int: String].Index` isn't Int. Its `Key` is

Reply:
And also this subscript doesn't make much sense for Dictionaries, since they `Key` subscript already returns optional value